jsapp/js/components/modalForms/projectSettings.es6

```diff
@@ -873,6 +873,10 @@ class ProjectSettings extends React.Component {
     const isSelfOwned = assetUtils.isSelfOwned(this.state.formAsset);
     const descriptionField = envStore.data.getProjectMetadataField('description');
 
+    const deleteButtonLabel = this.state.formAsset.deployment__submission_count > 0 ?
```

Comment:
I wanted to check this real quick, since it's possible for `deployment__submission_count` to be null when viewed by certain users (see [kpi#4739](https://github.com/kobotoolbox/kpi/pull/4739)). But thanks to the magic of JS, `null > 0` is false, so no problem here.

The actual delete button below has `isSelfOwned && this.props.context === PROJECT_SETTINGS_CONTEXTS.EXISTING` guarding it.

Reply:
Thanks. I changed it and moved the condition to the element. Since it may not be rendered it does make more sense to leave the conditional label inside it as well. 🙇🏻